lib/tasks/manage.rake

```diff
@@ -95,6 +95,26 @@ namespace :manage do
     end
   end
 
+  def update_business_finder_subscriptions
+    business_finder_facet_group_content_id = "52435175-82ed-4a04-adef-74c0199d0f46".to_s.freeze
+
+    primary_business_finder_subscriber_list = SubscriberList.where(title: "EU Exit guidance for your business or organisation")
+
+    business_finder_subscriber_lists = SubscriberList.find_by_links_value(business_finder_facet_group_content_id) - primary_business_finder_subscriber_list
+    business_finder_subscriber_list_ids = business_finder_subscriber_lists.each(&:id).to_a
```

Comment:
I _think_ this can be rewritten as:

```ruby
    business_finder_subscriber_list_ids = business_finder_subscriber_lists.map { &:id }
```

...but I might be wrong!

Reply:
ah! I have to use 'map' because I want it to return a new array with ids

Reply:
`map` does return a new array 👍